settings.gradle

```diff
@@ -149,6 +151,7 @@ include "support:cas-server-support-scim"
 include "support:cas-server-support-gua"
 include "support:cas-server-support-surrogate-api"
 include "support:cas-server-support-surrogate-authentication"
+include "support:cas-server-support-surrogate-authentication-couchb"
```

Comment:
`couchdb`

Reply:
Done... That one took me a while to find yesterday...